### ISSUE TITLE
Lint is failing

### ISSUE THREAD

**Issue by garethsb** [AMWA-TV/nmos-parameter-registers#52]
Seems to be because we use links from the Capabilities register entries to the related Flow and Sender Attributes registers using the directory name like:  `../flow-attributes/#colorspace`, `../sender-attributes/#st-2110-21-sender-type`

@peterbrightwell Any idea how to fix this?

At the moment the above links get rendered with no change, so the browser ends up at e.g.

https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#colorspace

Putting `README.md` or `index.html` or something in the URL just to please the linter doesn't seem like a good strategy...

```
capabilities/README.md
  122:67-122:[11](https://github.com/AMWA-TV/nmos-parameter-registers/actions/runs/3026668125/jobs/4870265685#step:5:12)6  warning  Link to unknown heading in `../flow-attributes`: `colorspace`. Did you mean `colorspace`                                missing-heading-in-file  remark-validate-links
  130:67-130:[12](https://github.com/AMWA-TV/nmos-parameter-registers/actions/runs/3026668125/jobs/4870265685#step:5:13)9  warning  Link to unknown heading in `../flow-attributes`: `transfer-characteristic`. Did you mean `transfer-characteristic`      missing-heading-in-file  remark-validate-links
  155:70-155:117  warning  Link to unknown heading in `../flow-attributes`: `bit-rate`. Did you mean `bit-rate`                                    missing-heading-in-file  remark-validate-links
  163:58-163:104  warning  Link to unknown heading in `../flow-attributes`: `profile`. Did you mean `profile`                                      missing-heading-in-file  remark-validate-links
  171:56-171:100  warning  Link to unknown heading in `../flow-attributes`: `level`. Did you mean `level`                                          missing-heading-in-file  remark-validate-links
  179:65-179:112  warning  Link to unknown heading in `../flow-attributes`: `sublevel`. Did you mean `sublevel`                                    missing-heading-in-file  remark-validate-links
  219:55-219:106  warning  Link to unknown heading in `../sender-attributes`: `bit-rate`. Did you mean `bit-rate`                                  missing-heading-in-file  remark-validate-links
  242:52-242:119  warning  Link to unknown heading in `../sender-attributes`: `packet-transmission-mode`. Did you mean `packet-transmission-mode`  missing-heading-in-file  remark-validate-links
  251:68-251:[13](https://github.com/AMWA-TV/nmos-parameter-registers/actions/runs/3026668125/jobs/4870265685#step:5:14)3  warning  Link to unknown heading in `../sender-attributes`: `st-2110-21-sender-type`. Did you mean `st-2110-21-sender-type`      missing-heading-in-file  remark-validate-links
```

**Comment by peterbrightwell** [comment]
@cristian-recoseanu and I discussed on today's ARG. Two options are:

- change those links to the required specs.amwa.tv location
- add `README.md` and remove it with yet another special case in `fix-links.sh` in `nmos-render-scripts`

**Comment by peterbrightwell** [comment]
We also considered restructuring the parameter registers repo to be more like the others. Or even split it into one repo per register...

**Comment by garethsb** [comment]
Resolved by #63 🤷‍♂️
